lib/shrinkwrap.js

```diff
@@ -183,8 +183,10 @@ const assertNoNewer = async (path, data, lockTime, dir = path, seen = null) => {
         await assertNoNewer(path, data, lockTime, child, seen)
       else if (ent.isSymbolicLink()) {
         const target = resolve(parent, await readlink(child))
-        const tstat = await stat(target).catch(() => null)
+        const tstat = await stat(target).catch(
+          /* istanbul ignore next - windows */ () => null)
```

Comment:
what's the windows issue here? just curious

Reply:
Windows tests fail if you try to create a symlink to a non-existent file and you're not an Admin and haven't set the flag to allow non-Admins to create symbolic links.  Easier to just skip that test on windows, but that means this line won't ever get covered.  Previously, it was getting hit because GH Actions does allow symlinks on windows for non-admins, but I try to keep my windows laptop in a more or less default state, so I found it there.

Reply:
ahhhh ok fair enough. so to get coverage there we'd have to have a test that mocks stat with something that rejects and verify that everything went about its business regardless, which being that the thing we're covering is `() => null` that seems pretty excessive

:+1: for just skipping coverage